app/enterprise/changelog.md

```diff
@@ -4,6 +4,127 @@ no_search: true
+  Contact your Kong sales representative for more information. 
+
+#### Core
+- Kong checks version compatibility between the control plane and any data planes to ensure 
+the data planes and any plugins have compatibility with the control plane in hybrid mode. 
+The sync is stopped if the major/minor version differ or if the installed plugin versions 
+differ between control plane and data plane nodes.
+- Kong now accepts UTF-8 characters in route and service names. Entities with a `name` field 
+now support UTF-8 characters.
+- Certificates now have `cert_alt` and `key_alt` fields to specify an alternative certificate 
+and key pair.
+- The Go pluginserver `stderr` and `stdout` are now written into Kong’s logs, allowing Golang’s 
+native `log.Printf()`. 
+  - Introduced support for multiple pluginservers. This feature is backwards-compatible with 
+  the existing single Go pluginserver.
+
+#### Plugin Development Kit (PDK)
+- Introduced a `kong.node.get_hostname` method that returns current node's host name.
+- Introduced a `kong.cluster.get_id` method that returns a unique ID
+  for the current Kong cluster. If Kong is running in DB-less mode
+  without a cluster ID explicitly defined, this method returns nil.
+  For Hybrid mode, all Control Planes and Data Planes belonging to the
+  same cluster returns the same cluster ID. For traditional database
+  type deployments, all Kong nodes pointing to the same database will
+  also return the same cluster ID.
+- Introduced a `kong.log.set_serialize_value`, which allows customizing
+  the output of `kong.log.serialize`.
+
+#### Plugins
+- The [HTTP Log](https://docs.konghq.com/hub/kong-inc/http-log/) (`http-log`) plugin 
+has been improved to allow you to add headers to the HTTP request, which will help you integrate 
+with many observability systems.
+- The [Key Authentication](https://docs.konghq.com/hub/kong-inc/key-auth/) (`key-auth`)
+plugin has two new configuration parameters: `key_in_header` and `key_in_query`. Both 
+are booleans and tell Kong whether to accept (true) or reject (false) passed in either 
+the header or the query string. Both default to “true.”
+- The [Request Size Limiting](https://docs.konghq.com/hub/kong-inc/request-size-limiting/) 
+(`request-size-limiting`) plugin has a new configuration `require_content_length` that 
+causes the plugin to ensure a valid Content-Length header exists before reading the request body.
+- The [Serverless Functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/) 
+(`serverless-functions`) plugin introduces a sandboxing capability, is *enabled* by default, 
+and where only Kong PDK, OpenResty `ngx` APIs, and Lua standard libraries are allowed.
+
+#### Configuration
+- `client_max_body_size` and `client_body_buffer_size`, that previously
+  hardcoded to 10m, are now configurable through `nginx_admin_client_max_body_size` 
+  and `nginx_admin_client_body_buffer_size`.
```

Comment:
We can link here to e.g. https://docs.konghq.com/2.3.x/configuration/#nginx_http_client_max_body_size

Reply:
> We can link here to e.g. https://docs.konghq.com/2.3.x/configuration/#nginx_http_client_max_body_size

Linked to both options in commit 37993b4